target/linux/ramips/image/mt7621.mk

```diff
@@ -1487,6 +1487,17 @@ define Device/unielec_u7621-06-64m
 endef
 TARGET_DEVICES += unielec_u7621-06-64m
 
+define Device/dual-q_h721
+  $(Device/dsa-migration)
+  $(Device/uimage-lzma-loader)
+  IMAGE_SIZE := 16064k
+  DEVICE_VENDOR := Dual-Q
+  DEVICE_MODEL := H721
+  DEVICE_VARIANT := 16M
```

Comment:
Are there any other variants?

Reply:
hi, no other variants。

Reply:
If there're no other variants, please merge the dtsi file into the dts file. (So only one dts and no dtsi)